homeassistant/components/fujitsu_fglair/config_flow.py

```diff
@@ -22,11 +23,20 @@
         vol.Required(CONF_EUROPE): bool,
     }
 )
+STEP_REAUTH_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_PASSWORD): str,
+    }
+)
 
 
 class FGLairConfigFlow(ConfigFlow, domain=DOMAIN):
     """Handle a config flow for Fujitsu HVAC (based on Ayla IOT)."""
 
+    def __init__(self) -> None:
+        """Initialize."""
+        self._reauth_data: dict[str, Any] = {}
```

Comment:
You can also just store the reauth_entry here so you don't have to fetch it later